featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -499,6 +501,18 @@ def test_training_window(pd_es):
                                               include_cutoff_time=False)
     prop_values = [5, 5, 2]
     dagg_values = [3, 2, 1]
+
+    assert (feature_matrix[property_feature.get_name()] == prop_values).values.all()
+    assert (feature_matrix[dagg.get_name()] == dagg_values).values.all()
+
+    # Case3. include_cutoff_time = False with single cutoff time value
```

Comment:
Can we add a case 4 for if `include_cutoff_time` is `True`?

Reply:
Added this additional test case.